lib/PrintAsClang/DeclAndTypePrinter.cpp

```diff
@@ -1949,8 +1949,9 @@ class DeclAndTypePrinter::Implementation
 
     if (outputLang == OutputLanguageMode::Cxx) {
       // FIXME: Documentation.
-      auto *getter = VD->getOpaqueAccessor(AccessorKind::Get);
-      printAbstractFunctionAsMethod(getter, /*isStatic=*/VD->isStatic());
+      // TODO: support read/modify accessors.
+      if (auto *getter = VD->getOpaqueAccessor(AccessorKind::Get))
+        printAbstractFunctionAsMethod(getter, /*isStatic=*/VD->isStatic());
       if (auto *setter = VD->getOpaqueAccessor(AccessorKind::Set))
```

Comment:
What would happen if a Swift property has a `_read` accessor and a setter? Could you please add a test just to make sure we don't crash?

Reply:
Could you help me come up with an example? In the code snippets I tried swift synthesized a getter. Seemingly the only way I could prevent that is to have a property with `~Copyable` type. But in that case, a setter would not compile, I need a `_modify` accessor. 

If there is any other way to prevent Swift from synthesizing a getter (despite me only spelling out a `_read` accessor) let me know.

Reply:
I was thinking of something like this:
```swift
struct Foo {
  private var x: Int = 123
  var computedVar: Int {
    _read { yield &x }
    set { x = newValue }
  }
}
```
Would Swift synthesize a getter in this case?

Reply:
I got this error:
```
20 |     _read { yield &x }
   |                   `- error: '&' may only be used to pass an argument to inout parameter
```

After removing the `&`, I still see `Foo::getComputedVar()` in the generated interop header, so I think the getter is being synthesized in this case.